andela/ah-titans-frontend
# #160318853 Create view articles

#### What does this PR do?
Implements view articles feature.

#### Description of tasks to be completed
Implements loading of all articles when a user navigates to the home page.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout ft-create-view-articles-160318853
```
```sh
$ cd ah-titans-frontend
```

> Install the dependencies with the command below.

```sh
$ npm install
```
>Then run the app with the command below.
```sh
$ npm start
```
> You should get to the home page (`http://localhost:4000/`)

### Relevant screenshots
> While the articles are loading

![image](https://user-images.githubusercontent.com/12577659/45423366-cf5d8980-b69b-11e8-8bb4-3630622db4d5.png)

> After the articles have loaded

![image](https://user-images.githubusercontent.com/12577659/45423804-1009d280-b69d-11e8-8949-88db615bfe89.png)


#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160318853](https://www.pivotaltracker.com/story/show/160318853)
